packages/default/scss/grid/_layout.scss

```
@@ -973,6 +973,36 @@
             display: none;
         }
     }
+
+    // Remove Grid scrollbar during built-in export
+    .k-pdf-export-shadow {
+        .k-grid {
+            float: left;
+            width: auto !important;
+        }
+
+         // Remove all sizes and scrolling
+        .k-grid,
+        .k-grid-content,
+        .k-grid-content-locked {
+            height: auto !important;
+            overflow: visible;
+        }
+
+        .k-grid-header-locked {
+            & + .k-grid-header-wrap,
+            & + .k-grid-content,
+            & + .k-grid-footer-wrap {
+                width: auto !important;
```

Consider adding sasslint disable line:
```suggestion
                width: auto !important; // sass-lint:disable-line no-important
```